<?xml version="1.0"?>
<launch>
  <arg name="nodelet_name" default="camera_nodelet_manager" />
  <arg name="independent_nodelet" default="true" />
  <arg name="headless" default="false" />
  <arg name="image_name" default="/camera/image" />

  <rosparam file="$(find jsk_uav_forest_perception)/param/ColorRegionDetection.yaml" command="load" />

  <group ns="camera" >
    <!-- nodelet manager -->
    <node type="nodelet" pkg="nodelet" name="$(arg nodelet_name)" args="manager" output="screen" if="$(arg independent_nodelet)"/>

    <!-- color region detector -->
    <node pkg="nodelet" type="nodelet" name="color_region_detection"
          args="load color_region_detection/ColorRegionDetector $(arg nodelet_name)" output="screen">
      <param name="image_topic_name" value="$(arg image_name)"/>
      </node>

  </group>

</launch>
